### ISSUE TITLE
Unable to call backend api with authorization header

### ISSUE THREAD

**Issue by asterd** [intive-opensource/adyen_flutter#5]
Hi,

I try to use your plugin, but my backend is secured through a bearer-token in authorization header. 
Since your plugin doesn't provide a way to specify custom header for backend calls, I'm unable to use it.
Is it possible to add a simple parameter where I can specify the baseUrl and also a custom http header?

Thanks a lot!

**Comment by jimykhan** [comment]
Hi,
did this plugin work for you. if yes any link that i can take some hints i want to implement adyen to my flutter project.
thanks

**Comment by asterd** [comment]
Yes and no. This plugin works, but is not updated with latest version of SDK and doesn't implement some features I need.
I've forked in in my own github and I use that version in production without problems.

**Comment by bennibau** [comment]
@asterd would you be so kind and create a PR for this repository so we can publish your changes?
